content/embeds/k8s-admission-webhook-cert.md

```diff
@@ -27,15 +27,15 @@
 
 4. Create a patch file for the Kubernetes validating webhook.
 
-    ```sh
-    cat > modified-webhook.yaml <<EOF
-    webhooks:
-    - name: redisenterprise.admission.redislabs
-      clientConfig:
-        caBundle: $CERT
-      admissionReviewVersions: ["v1beta1"]
-    EOF
-    ```
+```sh
+cat > modified-webhook.yaml <<EOF
+webhooks:
+- name: redisenterprise.admission.redislabs
+  clientConfig:
+    caBundle: $CERT
+  admissionReviewVersions: ["v1beta1"]
+EOF
+```
```

Comment:
This item doesn't have the same indentation as the others above it, and it kinda sticks out like a sore thumb. Same for the other page. All the code examples should have uniform indentation.